### ISSUE TITLE
Investigate consequences of hitting GitHub API Limit

### ISSUE THREAD

**Issue by kathy-phet** [phetsims/rosetta#416]

**Comment by jbphet** [comment]
In #410 a problem was logged where the interaction between Rosetta and the GitHub API was hitting a limit in the number of allowed interactions per hour.  This was addressed by adding code to monitor the number of API calls and throttle it back when Rosetta gets close to hitting this limit.

In a meeting today about next steps for Rosetta, @kathy-phet expressed concern about whether hitting this limit could potentially cause problems for other PhET-GitHub interactions, or even interactions for other CU users of GitHub, since we are using the CU enterprise plan for GitHub.  I looked into this a bit and I think we're fine here.  This limitation is imposed on a per-PAT (Personal Access Token) basis, and we use a unique PAT for Rosetta's interactions with GitHub.  Also, since the throttling code is in place, we shouldn't be hitting the limit at all anymore.

I have logged a separate issue about investigating more efficient Rosetta-GitHub API interaction, see https://github.com/phetsims/rosetta/issues/418.

@kathy-phet - I've investigated the consequences and I think we're safe.  Is there anything you'd like to know or have done on this before closing?
